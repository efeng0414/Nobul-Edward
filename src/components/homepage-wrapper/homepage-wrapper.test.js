import React from "react";
import renderer from "react-test-renderer";
import HomepageWrapper from "./index";

describe("HomepageWrapper", () => {
  const componentProps = {
    children: "children",
    backgroundImage: "backgroundUrl"
  };

  const componentJSX = <HomepageWrapper {...componentProps} />;
  describe("Component Snapshot", () => {
    const componentSnapshot = renderer.create(componentJSX).toJSON();
    it("should match existing snapshot", () => {
      expect(componentSnapshot).toMatchSnapshot();
    });
  });
});
