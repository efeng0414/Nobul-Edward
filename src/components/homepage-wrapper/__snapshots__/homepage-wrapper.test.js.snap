// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomepageWrapper Component Snapshot should match existing snapshot 1`] = `
<div
  className="background home-page "
  style={
    Object {
      "backgroundColor": "",
      "backgroundImage": "url(backgroundUrl)",
    }
  }
>
  <div
    className="background-content"
  >
    children
  </div>
</div>
`;
